src/ui/components/navigation/navigationcomponent.js

```diff
@@ -238,6 +239,11 @@ export default class NavigationComponent extends Component {
     }
   }
 
+  setParentUrl (parentUrl) {
```

Comment:
I'm sort of torn here. The reason we have this as a setter is for the Theme. Conceptually, though, I think this probably makes more sense as a config option for the Component. I can't really see a use-case where it's set more than once. If we made it part of the Component config, though, we couldn't get the Theme portion working, correct?

Reply:
Could this be a config option if it was a getter? so I was thinking `{ getParentUrl: () => /* fancy logic */ }`

Reply:
In general calling the component instance directly isn't common

Reply:
the iframe resizer is loaded async from the ANSWERS initialization and component creation portion. I don't think we can guarantee to get the parentUrl from the iframe's parent to put it as part of the component config at the time of adding/creating navigation component.

Reply:
I was thinking we could put parent URL in RuntimeConfig, and then have the the getParentUrl config option call RuntimeConfig, but I'm now realizing the nav component would still have to setState again to update the URLs. I'm fine with the current approach!